microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -519,6 +519,45 @@
         });
     }
 
+    function updateDatasetDetails(state){
+        $.ajax(
+            {
+                url: state.public_endpoint + "/dataset/" + state.dataset_input.value,
+                success: function(data)
+                {
+                    var info = data[state.dataset_input.value]
+                    if (info == null)
+                        return
+                    $("#dataset_title").text(info.title).removeClass("spinner-grow spinner-grow-sm");
+                    if (info.datatype == "WGS"){
+                        $("#dataset_datatype").text("whole genome sequencing, a technique that reads sequences from anywhere in the genomes of any of the bacteria in the gut.  With enough read coverage, you can even see the variation that exists between individuals of the same species. ").removeClass("spinner-grow spinner-grow-sm");
```

Comment:
"whole genome sequencing, a technique that reads sequences from anywhere in the genomes of any of the bacteria in the gut.  With enough read coverage, you can even see the variation that exists between individuals of the same species. "

To

"whole genome sequencing, a technique that produces a large number small of DNA sequences from all of the microbial genomes of in your sample.  With enough sequences you can even see the variation that exists between microbes of the same species. "